manifests/server/dependencies.pp

```diff
@@ -2,14 +2,23 @@
 #
 # @api private
 #
-# @author Tim Meusel <tim@bastelfrek.de>
-class hyperglass::server::dependencies {
+# @author Tim Meusel <tim@bastelfreak.de>
+class hyperglass::server::dependencies (
+  Boolean $manage_python = $hyperglass::server::manage_python,
+  Boolean $manage_gcc    = $hyperglass::server::manage_gcc,
+) {
   assert_private()
 
-  package { ['python3-pip', 'python3-devel', 'gcc']:
-    ensure => 'installed',
+  if $manage_python {
+    include hyperglass::python
```

Comment:
`require` instead of `include` here (and in several other places) would remove the need for the following two lines.

Reply:
thanks. fixed them